py/mpconfig.h

```diff
@@ -597,6 +597,14 @@
 #define MICROPY_READER_VFS (0)
 #endif
 
+#ifndef MICROPY_READER_VFS_BUFFER_SIZE
+#if MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EXTRA_FEATURES
+#define MICROPY_READER_VFS_BUFFER_SIZE (24 + 180)
+#else
+#define MICROPY_READER_VFS_BUFFER_SIZE (24 + 32)
```

Comment:
Maybe the 24 should remain a fixed constant in the `mp_reader_vfs_t` struct, since it pads it out to 32 bytes, which is a GC block on 32-bit systems.  Then this constant would be additional memory to allocate on top of the 24.

Reply:
That makes sense, I've pushed that change (also increasing it to 24 + 192 to keep it divisible by 32)